aether-kernel/aether/kernel/api/views.py

```diff
@@ -312,13 +313,39 @@ class MappingSetViewSet(MtViewSetMixin, viewsets.ModelViewSet):
     filter_class = filters.MappingSetFilter
     mt_field = 'project'
 
+    def destroy(self, request, pk=None):
```

Comment:
Could it be possible not to override the standard method and create a new one for this? The standard REST expects a 204 NO CONTENT response for the DELETE action... :woman_shrugging: